tests/query/parser/test_conditions.py

```diff
@@ -329,6 +329,12 @@
             ),
         ),
     ),  # Test that a duplicate IN condition is deduplicated even if the lists are in different orders.
+    (
+        [["group_id", "IS NULL", None]],
```

Comment:
It would be good the test the negative cases as well (unary operator with two operands, binary with one).

Reply:
done